files/en-us/web/javascript/reference/global_objects/regexp/flags/index.html

```diff
@@ -12,7 +12,7 @@
 ---
 <div>{{JSRef}}</div>
 
-<p>The <strong><code>flags</code></strong> property returns a string consisting of the <a href="/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#Advanced_searching_with_flags_2">flags</a> of the current regular expression object.</p>
+<p>The <strong><code>flags</code></strong> property returns a string consisting of the <a href="/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#Advanced_searching_with_flags">flags</a> of the current regular expression object.</p>
```

Comment:
```suggestion
<p>The <strong><code>flags</code></strong> property returns a string consisting of the <a href="/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#advanced_searching_with_flags">flags</a> of the current regular expression object.</p>
```
Almost! I had to lowercase it to make it jump. Does that work for you?